src/config.rs

```diff
@@ -24,6 +24,10 @@ fn default_bold() -> bool {
     false
 }
 
+fn default_italic() -> bool {
+    false
+}
+
```

Comment:
What is the purpose of this function? 

It may be better to just use a `const ITALICS_DEFAULT: bool = false`

Or just inline in the `RawStyle` annotation. `#[serde(default = false)]`

Reply:
I looked at how other styles were implemented (like bold and underline) and those were done the same way, I was not looking to make changes to unrelated stuff

Reply:
Also the project does not build when inlining the default value, see: https://serde.rs/field-attrs.html#default--path